k8s-fluentd/main.tf

```diff
@@ -0,0 +1,32 @@
+locals {
+  # this should be the default calculated name anyway, but we want to enforce it just to be sure
+  service_account_name = var.release == "fluentd" ? "fluentd" : "${var.release}-fluentd"
+
+  # extract data about kubernetes cluster
+  cloud            = var.kubernetes_cluster.specs.kubernetes.cloud
+  k8s_distribution = var.kubernetes_cluster.specs.kubernetes.distribution
+
+  helm_values = {
+    serviceAccount = {
+    name = local.service_account_name
+    }
+    labels = var.md_metadata.default_tags
+    plugins = ["fluent-plugin-opensearch"]
```

Comment:
FYI, ultimately we aren't gonna want this hardcoded for opensearch. If this is just to get it out the door and testing fast, thats fine. But `cloudwatch`[https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-EKS-logs.html], opensearch, `stackdriver`[https://cloud.google.com/stackdriver/docs/solutions/gke/managing-logs#custom_agents], elasticsearch, etc all will be using fluentd to get the logs where they need to be.

Reply:
yeah I can back this out and it can be up to the bundle to install necessary plugins based on what logging destination we have.